test/scpi.py

```diff
@@ -185,7 +116,6 @@ def dcxo_calibrate(classname, iio_uri):
 
     sdr._set_iio_dev_attr("dcxo_tune_coarse", coarse, sdr._ctrl)
     sdr._set_iio_dev_attr("dcxo_tune_fine", fine, sdr._ctrl)
-
     while 1:
         current_frq = get_freq(instr)
```

Comment:
Any scenario when this would never exit?

Reply:
I don't think there is any